GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -52,42 +52,42 @@ private func parseHexString(hexString: String) throws -> UInt32 {
 // MARK: - Text File Parser
 
 public final class ColorsTextFileParser: ColorsFileParser {
-  public private(set) var colors = [String:UInt32]()
+  public private(set) var colors = [String : UInt32]()
```

Comment:
Ditto (note that some of those might be from me and my old code-style, not all from changes in this PR? But would be nice to uniform this to match SwiftLint's rules anyway)